mimesis/providers/transport.py

```diff
@@ -56,14 +58,13 @@ def airplane(self, model_mask: str = '###') -> str:
         plane = self.random.choice(AIRPLANES)
         return '{} {}'.format(plane, model)
 
-    def vehicle_registration_code(self, allow_random: bool = True) -> str:
+    def vehicle_registration_code(self, locale: Optional[str] = None) -> str:
         """Get vehicle registration code of country.
 
-        :param allow_random: If False return only code
-            for current locale (country).
+        :param locale: Registration code for locale (country).
         :return: Vehicle registration code.
         """
-        if not allow_random:
-            return VRC_BY_LOCALES[self.locale]
+        if locale:
```

Comment:
Why do we need this change? What's your motivation?

Reply:
To make `Code` locale-independent, because only this method needs locale code in this provider. So, it better just pass locale code to this method, instead of making this provider locale-dependent.